mkdocs.yml

```diff
@@ -15,6 +15,7 @@ nav:
   - Analysis: analysis.md
   - Telegram: telegram.md
   - First-time users: walkthrough.md
+  - Installing on a Pi: raspberry.md
```

Comment:
Instead of making this a new page, can you please make this a section on the 'installation' page?

Reply:
Sure thing